tests/behaviour/steps/concept/type_/relation_type.rs

```diff
@@ -176,14 +137,20 @@ pub async fn relation_role_unset_override(
             )
             .unwrap()
             .unwrap();
-        let res = relates.unset_override(Arc::get_mut(&mut tx.snapshot).unwrap(), &tx.type_manager, &tx.thing_manager);
+        let res =
+            relates.unset_specialise(Arc::get_mut(&mut tx.snapshot).unwrap(), &tx.type_manager, &tx.thing_manager);
         may_error.check_concept_write_without_read_errors(&res);
     });
 }
 
 #[apply(generic_step)]
-#[step(expr = r"relation\({type_label}\) get roles {contains_or_doesnt}:")]
-pub async fn relation_roles_contain(context: &mut Context, type_label: Label, contains: ContainsOrDoesnt, step: &Step) {
+#[step(expr = r"relation\({type_label}\) get relates {contains_or_doesnt}:")]
```

Comment:
If I say `get role(...)`, I believe that I am talking to the root role for my relation (`relation:role` is the label of this `role`, not `superrelation:role`!)
And for explicity of checks, you cannot say
`relation(subrelation) get role(superrole)` if `superrole`'s label is `superrelation:superrole` in Concept API tests.
Instead, you have to work with `relation(superrelation) get role(superrole)`!

At the same time, we want to check what's inside relates, so now we have `get relates contains`, etc.